arviz/stats/stats.py

```diff
@@ -611,6 +611,18 @@ def psislw(log_weights, reff=1.0):
         Smoothed log weights
     kss : array
         Pareto tail indices
+
+    Examples
+    --------
+    Get Pareto smoothed importance sampling (PSIS):
```

Comment:
Get Pareto smoothed importance sampling (PSIS) log weights.

We should also add a reference, i.e. https://arxiv.org/abs/1507.02646 (see `ess` function for an example on the References section)